tests/unit/test_pipelines.py

```diff
@@ -84,18 +84,20 @@ def test_prepare_payload(
     result = pipeline._prepare_payload(spider)
 
     # acquisition_source has a timestamp
-    result['results_data'][0]['acquisition_source'].pop('datetime')
+    record_result = result['results_data'][0]['record']
+    record_result['acquisition_source'].pop('datetime')
```

Comment:
Can we rather use `freezegun` instead of this hack?

Reply:
Like it :+1: